### PR TITLE
Handle defaults better for mde.keep_centered setting

### PR DESCRIPTION
**Pull request by felixhao28**
closes #261 #260

### CODE REVIEW

File: distraction_free_mode.py [hunk 1]
```diff
@@ -21,11 +21,11 @@ def on_modified(self, view):
             return False
 
         if on_distraction_free():
-            if view.settings().get("mde.distraction_free_mode").get("mde.keep_centered") is False:
+            if not view.settings().get("mde.distraction_free_mode", {})
```
Comment (maliayas) [inline distraction_free_mode.py]: Dropping `\\` from the end of this line gives syntax error.

File: distraction_free_mode.py [hunk 2]
```diff
@@ -21,11 +21,11 @@ def on_modified(self, view):
...
+                              .get("mde.keep_centered", True):
                 return False
 
-        else:
-            if view.settings().get("mde.keep_centered") is False:
+        elif not view.settings().get("mde.keep_centered", False):
                 return False
```
Comment (maliayas) [inline distraction_free_mode.py]: This line is too much indented.

**Comment by maliayas** [comment]
By the way please push directly v.X.X.X branch of **SublimeText-Markdown/MarkdownEditing**. Otherwise, by pushing first to your own repo and then merging into here, commit logs easily becomes unreadable. This is what my git software displays for the latest changes:

ekran goruntusu 2015 04 16 22_47_42

It is hard to read and doesn't give credit to suor's commit. Commit diffs of several commits there really hurts brain. This is the github network graph of the same commits:

ekran goruntusu 2015 04 16 23_07_53

Another note: if we want to edit someone's changes, let's first cherrypick (or whatever) his commit and then create another commit for our modification. Not do the both in one commit.

After I rewrote the history, github graph became this:

ekran goruntusu 2015 04 16 23_21_32

**Comment by felixhao28** [comment]
I was a bit careless when editing the code. Should we revert this?

I usually work on my branch for new hotfixes. So when a change is made and ready to be merged into dev branch, instead of creating a PR, I create a new commit including the changes on dev branch and push it. Is that right?

**Comment by maliayas** [comment]
I reverted already, no problem.

Using your own repo and then merging into upstream without a pr is ok. The only thing you shouldn't do is not _merging_ into your repo from upstream in the middle of your development because this creates zigzag merges when you merge back into upstream at the end.

Maybe that became by merging several pr by ui, I don't blame you.

You are doing pretty good to the project.  Thanks for it.

**Pull request merged**